### DISCUSSION TITLE
Relation column not keeping order

### DISCUSSION THREAD

**Discussion by synapse** [pocketbase/pocketbase#478] [Q&A] [answered]
Hey guys, is there any way to keep the relation ids in the order they where inserted?
It's a really nice feature since it can be also expanded, but I cannot make it keep the order which is important.

**Comment by ganigeorgiev** [comment, marked as answer]
@synapse This should be already supported, at least when sending the data via the API (or SDK).

The relation ids are not preserved when visualized in the admin, because the select options are always populated and ordered by `created`. This will be fixed together with #100, but for now it is a low priority.

**Reply by synapse** [reply]
Hey @ganigeorgiev, yup tried it via the API call and indeed the order it's kept which is good. 
Thanks for the quick answer.

**Reply by ganigeorgiev** [reply]
@synapse Just to let you know that the order in the select dropdown was fixed in the latest [v0.7.4 release](https://github.com/pocketbase/pocketbase/releases/tag/v0.7.4).
